src/oop_ext/interface/_interface.py

```diff
@@ -393,7 +451,7 @@ def _IsImplementationFullChecking(class_or_instance, interface):
         return True
 
 
-def _IsInterfaceDeclared(class_, interface):
+def _IsInterfaceDeclared(class_: Type, interface: Type[Interface]) -> bool:
```

Comment:
Line 463 is checking ` if class_ is None`

Reply:
Done, thanks!